src/orion/benchmark/assessment/paralleladvantage.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Parallel Advantage Assessment
+=============================
+"""
+
+from collections import defaultdict
+
+from orion.benchmark.assessment.base import BenchmarkAssessment
+from orion.executor.base import executor_factory
+from orion.plotting.base import durations, parallel_advantage, regrets
+
+
+class ParallelAdvantage(BenchmarkAssessment):
+    """
+    Evaluate the advantage to run experiment in multiple parallel workers.
+
+    Evaluate the average performance (objective value) for each search algorithm
+    at different time steps (trial number).
+    The performance (objective value) used for a trial will the best result until the trial.
+
+    Parameters
+    ----------
+    task_num: int
+        Number of experiment to run for each number of workers.
+    executor: str
+        Name of orion worker exeuctor.
+    n_workers: list
+        List of intergers for number of workers for each experiment.
+    executor_config: dict
+        Parameters for the corresponding executor.
+    """
+
+    def __init__(
+        self, task_num=1, executor=None, n_workers=[1, 2, 4], **executor_config
+    ):
+
+        super(ParallelAdvantage, self).__init__(
+            task_num=task_num * len(n_workers),
+            executor=executor,
+            n_workers=n_workers,
+            **executor_config
+        )
+        self.executor_name = executor
+        self.executor_config = executor_config
+        self.n_workers = [n_worker for n_worker in n_workers for _ in range(task_num)]
+
+    def analysis(self, task, experiments):
```

Comment:
There are no unittests to verify how n_workers and task_num, nor that passing the executor config is used correctly. There are also no unittests for the analysis part, to verify that the figures return match number of workers and number of repetitions as expected.

Reply:
Done, thanks!